### PR TITLE
Documentation for PyPi uploads

### LINKED ISSUE #144: PyPi Upload Docs
* Update version in setup.py
* follow Peter Dowds instructions http://peterdowns.com/posts/first-time-with-pypi.html

### PR DESCRIPTION
**Pull request by matthewrsj**
fixes #144 

## Changes in this PR.

- [X] Add documentation for submitting Pymesync to PyPi

## Testing this PR.

```
(venv) $ cd docs
(venv) $ make html
(venv) $ <your-favorite-web-browser> build/html/index.html
```

You can review the content directly on the web page that is opened.

@osuosl/devs

### CODE REVIEW

**Comment by Kennric** [comment]
I like that, also nice to have links to all the relevant docs on tagging, pypi, etc. +1

File: docs/source/developer.rst [hunk 1]
```diff
@@ -131,3 +131,75 @@ Documenting Changes
...
+
+When new features are added or bugs are fixed it is necessary to push Pymesync
+to `PyPi`_ so users can ``pip install`` those changes. This can only be done by
+owners and maintainers of Pymesync, listed below. Email support@osuosl.org if
+you believe a new version is warranted.
+
+====================== ==========
+Developer              Role
+====================== ==========
+Matthew Johnson (mrsj) Owner
```
Comment (ElijahCaine) [inline docs/source/developer.rst]: You should note that the names (in parenthesis) are nicks on irc.freenode.net or  include GH usernames.

File: docs/source/developer.rst [hunk 2]
```diff
@@ -131,3 +131,75 @@ Documenting Changes
...
+     [... upload to pypi success ...]
+
+   .. note::
+
+     ``pypi`` is a configuration set in ``.pypirc`` from step 1.
+
+   Now visit https://pypi.python.org/pypi to make sure the version is up to
+   date.
+
+#. Inform your users that Pymesync has been updated.
```
Comment (ElijahCaine) [inline docs/source/developer.rst]: Is there a set way to do this?
Rephrasing it to be something like *Inform pymesync users that there has been an update in whatever way is the current standard.*, or something like that.

**Comment by ElijahCaine** [comment]
Looks pretty good. Citations are helpful and it's concise. There's a few assumptions but that's my only feedback. +1 pending that being addressed.

File: docs/source/developer.rst (hunk 2 above)
Comment (matthewrsj) [inline docs/source/developer.rst]: My set way of doing this:

```
10:26:17   mrsj | resistortwister, subnomo: new pymesync version is up on PyPi
```

**Comment by ElijahCaine** [comment]
LGTM +1

**Comment by Kennric** [comment]
+1

**Pull request merged**